data/application.css

```diff
@@ -22,5 +22,51 @@ textview.terminal {
 
 textview.terminal selection {
     background-color: #93a1a1;
-    color: #252e32; 
+    color: #252e32;
+}
+
+.partition {
+    border-style: solid;
+    border-right-width: 1px;
+}
+
+.disk-bar {
+    border-style: solid;
+    border-width: 1px;
+}
+
+.ext2, .ext3, .ext4 {
+    background-color: rgb(114, 147, 203);
```

Comment:
Where are these colors coming from? They do not pass any sort of contrast check. We should be sure to set all light colors where our foreground is more likely to contrast, or also be sure to set the foreground color to something that contrasts better.

Reply:
Colors were selected at random

Reply:
Can we use colors from the palette? :) https://elementary.io/docs/human-interface-guidelines#color